goal_src/jak2/build/all_objs.json

```diff
@@ -0,0 +1,2252 @@
+["spatial-hash",                               "spatial-hash",               3, ["ENGINE", "GAME"], "engine/data"],
+["actor-hash",                                 "actor-hash",                 3, ["ENGINE", "GAME"], "engine/data"],
+["merc-death",                                 "merc-death",                 3, ["ENGINE", "GAME"], "engine/gfx/merc"],
+["water-flow",                                 "water-flow",                 3, ["ENGINE", "GAME"], "engine/gfx/water"],
+["water-h",                                    "water-h",                    3, ["ENGINE", "GAME"], "engine/gfx/water"],
+["camera",                                     "camera",                     3, ["ENGINE", "GAME"], "engine/camera"],
+["cam-interface",                              "cam-interface",              3, ["ENGINE", "GAME"], "engine/camera"],
+["cam-master",                                 "cam-master",                 3, ["ENGINE", "GAME"], "engine/camera"],
+["cam-states",                                 "cam-states",                 3, ["ENGINE", "GAME"], "engine/camera"],
+["cam-states-dbg",                             "cam-states-dbg",             3, ["ENGINE", "GAME"], "engine/camera"],
+["cam-combiner",                               "cam-combiner",               3, ["ENGINE", "GAME"], "engine/camera"],
+["cam-update",                                 "cam-update",                 3, ["ENGINE", "GAME"], "engine/camera"],
+["vol-h",                                      "vol-h",                      3, ["ENGINE", "GAME"], "engine/geometry"],
+["cam-layout",                                 "cam-layout",                 3, ["ENGINE", "GAME"], "engine/camera"],
+["cam-debug",                                  "cam-debug",                  3, ["ENGINE", "GAME"], "engine/camera"],
+["cam-start",                                  "cam-start",                  3, ["ENGINE", "GAME"], "engine/camera"],
+["process-drawable",                           "process-drawable",           3, ["ENGINE", "GAME"], "engine/draw"],
+["ambient",                                    "ambient",                    3, ["ENGINE", "GAME"], "engine/ambient"],
+["speech",                                     "speech",                     3, ["ENGINE", "GAME"], "engine/sound"],
+["region",                                     "region",                     3, ["ENGINE", "GAME"], "engine/level"],
+["fma-sphere",                                 "fma-sphere",                 3, ["ENGINE", "GAME"], "engine/anim"],
+["script",                                     "script",                     3, ["ENGINE", "GAME"], "engine/util"],
+["generic-obs",                                "generic-obs",                3, ["ENGINE", "GAME"], "engine/game"],
+["lightning",                                  "lightning",                  3, ["ENGINE", "GAME"], "engine/gfx"],
+["carry-h",                                    "carry-h",                    3, ["ENGINE", "GAME"], "engine/game"],
```

Comment:
seems to be specific to the mech, maybe `engine/target/mech`?

Reply:
i was thinking it might be related to carrying lurkers on zoomers too? but that's probably in a vehicle specific file